### PR TITLE
Create announcements.md

### PR DESCRIPTION
**Pull request by OzGav**

### CODE REVIEW

File: docs/integration/announcements.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Announcements
+description: Details of the announcements functionality  
+---
+
+# Announcements
+
+Announcements are either Text-to-Speech or audio files that are sent to the MA players via the [`tts.xxxxxx`](https://www.home-assistant.io/integrations/tts/) or [`mass.play_announcement`](../faq/massannounce.md) HA actions.
```
Comment (marcelveldt) [inline docs/integration/announcements.md:8]:
```suggestion
Announcements are either Text-to-Speech or audio files that are sent (from Home Assistant) to the MA players via the [`tts.xxxxxx`](https://www.home-assistant.io/integrations/tts/) or [`mass.play_announcement`](../faq/massannounce.md) HA actions. 
```

File: docs/integration/announcements.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## MA Player Support
+
+If a player does not support native announcements, MA will take care of it by stopping any playing music, adjust the volume (as configured), play the announcement, and then restore the previous state.
+
+!!! note
+    The MA announcement feature will ONLY work reliably if the player reports the state (e.g. playing, paused, idle) and the progress report (elapsed time) correctly. If any of those are not updated correctly then announcements will work poorly.
+
+ESPHome and Snapcast are known to be unreliable with state updates so expect issues with those player types. The same applies to some HA media players.
+
+If a player does not resume after playing an announcement, check the HTTP profile settings. `Fixed content length` may need to be used. For example, (the current) ESPHome devices have this issue.
```
Comment (marcelveldt) [inline docs/integration/announcements.md:43]:
```suggestion
If a player does not resume after playing an announcement, or it takes a very long time, check the HTTP profile settings. `Fixed content length` may need to be used. For example, (the current) ESPHome devices have this issue.
```

File: docs/integration/announcements.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Currently the only (known) player type that supports native announcements is Sonos S2 speakers.  Hopefully more player types will follow.
+
+## MA Player Support
+
+If a player does not support native announcements, MA will take care of it by stopping any playing music, adjust the volume (as configured), play the announcement, and then restore the previous state.
+
+!!! note
+    The MA announcement feature will ONLY work reliably if the player reports the state (e.g. playing, paused, idle) and the progress report (elapsed time) correctly. If any of those are not updated correctly then announcements will work poorly.
+
+ESPHome and Snapcast are known to be unreliable with state updates so expect issues with those player types. The same applies to some HA media players.
```
Comment (marcelveldt) [inline docs/integration/announcements.md:41]: Can we make this sound less harsh ?
For Snapcast I know @SantiagoSotoC is working on it to imrpove it. Also on the ESPHome side there is a new media player component in the making

File: docs/integration/announcements.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+If a player does not resume after playing an announcement, check the HTTP profile settings. `Fixed content length` may need to be used. For example, (the current) ESPHome devices have this issue.
+
+## Group Behaviour
+
+If an announcement is sent to a Native Google Cast Group, Sync Group or a Universal Group, the announcement will be played to ALL child players. The preference to (not) use a pre-announcement sound is defined on the group player. The volume setting of the announcement is defined per player. If one of the child players was already playing, playback will be interrupted and NOT resumed. Bear that in mind with automations and target announcements to the individual players or just restart individual player playback if/when needed. Google Cast groups should play the announcement in sync.
```
Comment (marcelveldt) [inline docs/integration/announcements.md:47]:
```suggestion
If an announcement is sent to a native Google Cast Group, Sync Group or a Universal Group, the announcement will always be played to ALL child players. The preference to (not) use a pre-announcement sound is defined on the group player in this case. The volume setting of the announcement is defined per player. If one of the child players was already playing, playback will be interrupted and NOT resumed. Bear that in mind with automations and target announcements to the individual players or just restart individual player playback if/when needed. Google Cast groups should play the announcement in sync.
```

File: docs/integration/announcements.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+If an announcement is sent to a group of players that are manually synced, the announcement will be played to all childplayers (in sync)
```
Comment (marcelveldt) [inline docs/integration/announcements.md:49]:
```suggestion
If an announcement is sent to a group of players that are manually synced, the announcement will be played to all childplayers (in sync).
```

File: docs/integration/announcements.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+If an announcement is sent to a Sync Group (temporary or permananent) of which ALL child players support native announcements, the native functionality will be used to playback the announcement and MA will ensure the announcement is started at the same (millisecond precise) time so all players should play the announcement in sync.
```
Comment (marcelveldt) [inline docs/integration/announcements.md:51]:
```suggestion
If an announcement is sent to a Sync Group (temporary or permananent) of which ALL child players support native announcements, the native functionality will be used to playback the announcement and MA will ensure the announcement is started at (more or less) the same time so all players should play the announcement in sync.
```

**Review by marcelveldt — APPROVED** [review]

File: docs/integration/announcements.md (hunk 3 above)
Comment (OzGav) [reply docs/integration/announcements.md:41]:
```suggestion
ESPHome and Snapcast can be unreliable with state updates so issues may occur with those player types. The same applies to some HA media players. Future updates of those platforms should improve performance.
```

**Pull request merged**